beswifty.com.website.json

```diff
@@ -0,0 +1,24 @@
+{
+   "providerId":"beswifty.com",
+   "providerName":"Beswifty",
+   "serviceId":"website",
+   "serviceName":"Swifty Site",
+   "version": 4,
+   "logoUrl":"https://beswifty.com/wp-content/uploads/2019/12/Swifty-Logo_Tagline.svg",
+   "description":"Enables a domain to work with Swifty Sites",
+   "syncRedirectDomain":"*.beswifty.com",
```

Comment:
The specification does not foresee use of wildcards here. Multiple domains are possible though.
> When present, this is a comma separated list of domain names for which redirects must be sent to after applying a template for the synchronous flow.